### PR TITLE
Making Content Great Again batch 7

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

##related issue: 
https://github.com/demisto/etc/issues/23712

## Description
fixing content files so that they pass validation- batch

### CODE REVIEW

**Comment by hod-alpert** [comment]
validation fails with `You have added an image in the yml file, please update the package Packs/Legacy/Integrations` will be resolved next release because of this PR https://github.com/demisto/demisto-sdk/pull/422

File: Packs/GoogleVault/TestPlaybooks/playbook-Google_Vault_-_Search_And_Display_Results_test.yml [hunk 1]
```diff
@@ -82,7 +83,7 @@ tasks:
       searchMethod:
         simple: Specific Accounts(requires emails argument)
       startTime:
-        simple: 2019-07-10T00:00:00.000000Z
```
Comment (Itay4) [inline Packs/GoogleVault/TestPlaybooks/playbook-Google_Vault_-_Search_And_Display_Results_test.yml:85]: does the playbook still pass with this change?

File: Packs/Legacy/Integrations/integration-MISP.yml [hunk 2]
```diff
@@ -666,6 +666,8 @@ script:
       description: Hash of the file to query. Supports MD5, SHA1 and SHA256. Notice         that
         you can pass comma-separated multiple values to efficiently retrieve multiple
         responses.
+      default: true
+      isArray: true
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-MISP.yml:670]: need to check if `isArray` is actually supported in the code

File: Packs/Legacy/Integrations/integration-Mimecast-Auth.yml [hunk 3]
```diff
@@ -35,7 +35,7 @@ configuration:
   required: false
 - display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Mimecast-Auth.yml:38]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Mimecast.yml [hunk 4]
```diff
@@ -28,7 +28,7 @@ configuration:
   required: true
 - display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Mimecast.yml:31]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-PhishMe.yml [hunk 5]
```diff
@@ -26,12 +26,12 @@ configuration:
   required: true
 - display: Use system proxy settings
   name: proxy
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-PhishMe.yml:29]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-PhishMe.yml [hunk 6]
```diff
@@ -26,12 +26,12 @@ configuration:
...
   type: 8
   required: false
 - display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "false"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-PhishMe.yml:34]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Pwned.yml [hunk 7]
```diff
@@ -10,7 +10,7 @@ description: Deprecated. Use the Have I Been Pwned? V2 integration instead.
 configuration:
 - display: Use system proxy settings
   name: proxy
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Pwned.yml:13]:
```suggestion
  defaultvalue: ""
```

File: Packs/Legacy/Integrations/integration-Pwned.yml [hunk 8]
```diff
@@ -345,4 +345,4 @@ script:
   runonce: false
 deprecated: true
 tests:
-  - Pwned test
+  - No tests
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Pwned.yml:348]: why did you change this?

File: Packs/Legacy/Integrations/integration-jira.yml [hunk 9]
```diff
@@ -57,24 +57,22 @@ configuration:
   required: false
 - display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-jira.yml:60]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-jira.yml [hunk 10]
```diff
@@ -57,24 +57,22 @@ configuration:
...
   type: 8
   required: false
 - display: Use system proxy settings
   name: proxy
-  defaultvalue: "false"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-jira.yml:65]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-opswat-metadefender.yml [hunk 11]
```diff
@@ -22,7 +22,7 @@ configuration:
   required: false
 - display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-opswat-metadefender.yml:25]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-secdo.yml [hunk 12]
```diff
@@ -19,14 +19,14 @@ configuration:
   defaultvalue: ""
   type: 4
   required: true
-- display: Trust any certificate (insecute)
+- display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "true"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-secdo.yml:24]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-secdo.yml [hunk 13]
```diff
@@ -19,14 +19,14 @@ configuration:
...
   type: 8
   required: false
 - display: Use system proxy settings
   name: proxy
-  defaultvalue: "false"
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-secdo.yml:29]:
```suggestion
```

File: Tests/conf.json [hunk 14]
```diff
@@ -2825,12 +2837,10 @@
         "IsItPhishing": "No instance",
         "Verodin": "No instance",
         "EasyVista": "No instance",
-        "Pipl": "No instance",
         "Moloch": "No instance",
         "Twilio": "No instance",
         "Zendesk": "No instance",
         "GuardiCore": "No instance",
-        "Nessus": "No instance",
```
Comment (Itay4) [inline Tests/conf.json:2833]: are you synced with @ShahafBenYakir on this ?

File: Packs/GoogleVault/TestPlaybooks/playbook-Google_Vault_-_Search_And_Display_Results_test.yml (hunk 1 above)
Comment (hod-alpert) [reply Packs/GoogleVault/TestPlaybooks/playbook-Google_Vault_-_Search_And_Display_Results_test.yml:85]: didn't see that one, reverting

File: Packs/Legacy/Integrations/integration-MISP.yml (hunk 2 above)
Comment (hod-alpert) [reply Packs/Legacy/Integrations/integration-MISP.yml:670]: reverting

File: Packs/Legacy/Integrations/integration-Pwned.yml (hunk 8 above)
Comment (hod-alpert) [reply Packs/Legacy/Integrations/integration-Pwned.yml:348]: The test is not registered in conf.json and the integration is deprecated

File: Tests/conf.json (hunk 14 above)
Comment (hod-alpert) [reply Tests/conf.json]: Reverted

**Comment by Itay4** [comment]
@hod-alpert please resolve conflicts here

**Comment by Itay4** [comment]
@hod-alpert please check that as well
image

**Comment by Itay4** [comment]
build fails on rn for deprecated pack
merging
